internal/security/proxy/config.go

```diff
@@ -45,6 +45,30 @@ type KongUrlInfo struct {
 	ApplicationPortSSL int
 }
 
+// ServiceInfo provides the host and port of another service in the eco-system.
+type ServiceInfo struct {
+	// Name of the service
+	Name string
+	// Host is the hostname or IP address of a service.
+	Host string
+	// Port defines the port on which to access a given service
+	Port int
+	// Protocol indicates the protocol to use when accessing a given service
+	Protocol string
+}
+
+func (cfg *ConfigurationStruct) GetServices() map[string]ServiceInfo {
```

Comment:
also since these functions are exported, they should have a short doc comment. even though they are in the internal package, having doc comments is still a really useful thing to have

Reply:
Added comment :)